client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -251,7 +251,7 @@ const SiteSetupList = ( {
 				<CardHeading>{ translate( 'Site setup' ) }</CardHeading>
 				<ul className="site-setup-list__list">
 					{ tasks.map( ( task ) => {
-						const enhancedTask = getTask( task );
+						const enhancedTask = getTask( task, { userEmail } );
```

Comment:
Could you please elaborate on this change?

Reply:
Here I am destructuring and including `userEmail ` as a parameter while calling the `getTask` function. Now, within the `getTask` function, under the conditional check of `task.id`, if it is `CHECKLIST_KNOWN_TASKS.EMAIL_VERIFIED` then we use/display the user email in the Title and the description.